docs/concepts/why-filecoin.md

```diff
@@ -0,0 +1,79 @@
+
+## Content addressing
+
+Files are referred to by the data they contain, not by fragile identifiers such as URLs. Files remain available no matter where they are hosted or who they are hosted by. When a file becomes popular it can be quickly distributed by swarms of computers instead of relying on a central computer which easily becomes overloaded by network traffic.
+
+When multiple users store the same file (and choose to make the file public by not encrypting it), everyone who wants to download the file benefits from Filecoin keeping it available. No matter where a file is downloaded from, downloaders can verify they have received the correct file and that it is intact.
+
+## Content distribution network
+
+Retrieval miners are computers that have good network connections to lots of users who want to download files. By prefetching popular files and distributing them to nearby users, retrieval miners are rewarded for making network traffic flow smoothly and files download quickly.
+
+Filecoin’s retrieval market incentives the creation of an open, global, self-organized content distribution network. Filecoin bypasses the need for written agreements between every ISP and every content provider in order for content distribution to work. Instead, Filecoin makes its content distribution network available to anyone who wants to join, without needing to seek permission.
+
+## Single protocol
+
+Applications implementing Filecoin can store their data on any miner using the same protocol. There isn’t a different API to implement for each provider. Applications wishing to support several different providers aren’t limited to the lowest-common-denominator set of features supported by all their providers.
+
+## No lock-in
+
+Migrating to a different storage provider is made easier because they all offer the same set of services and API. Users aren’t locked in to providers because they rely on a particular feature of the provider. In addition, files are content-addressed, which enables them to be transferred directly between miners without the user having to download and re-upload the files.
+
+Traditional cloud storage providers lock users in by making it cheap to store files but expensive to retrieve them again. Filecoin avoids this by facilitating a retrieval market where miners compete to give users their files back as fast as possible, at the lowest possible price. 
+
+## Open source code
+
+The code that runs both clients and storage providers is open source. Storage providers don’t have to develop their own software for managing their infrastructure. Everyone benefits from improvements made to Filecoin’s code.
+
+## Active community
+
+Filecoin has an active community of contributors to answer questions and help newcomers get started. There is an open dialog between users, developers and storage providers. If you need help, you will likely be able to reach the person who designed or built the system in question.
```

Comment:
Link to community section so people can find the appropriate forums?

Reply:
Done!